kuma/landing/templates/landing/homepage.html

```diff
@@ -44,13 +44,35 @@
         <h1>{{ _('Shared knowledge <span>for the Open Web</span>') }}</h1>
     {% endif %}
 
-  <form id="home-search-form" action="{{ url('search') }}" method="get" class="search">
-    <div class="home-search-form search-form">
-      <label for="home-q" class="offscreen">{{ _('Search') }}</label>
-      <input class="search-input" type="search" id="home-q" name="q" placeholder="{{ _('Search the docs') }}" />
-      <button type="submit" class="offscreen">{{ _('Search') }}</button>
-    </div>
-  </form>
+    {% if waffle.flag('search_suggestions') %}
+        <form id="home-search-form" action="{{ url('search') }}" method="get" class="search nodisable">
+            <div class="home-search-form search-form">
+                <div class="search-filter-wrapper">
+                    <div id="filters-right" class="filter-arrow">
+                        <i class="icon-chevron-left" aria-hidden="true"></i>
+                    </div>
+                    <div class="filters"></div>
+                    <div id="filters-left" class="filter-arrow">
```

Comment:
This and filters-right should probably be buttons not <divs>, OR we can update the js to add tabindex, aria roles, and an event listener... I think buttons is easier ;)